### DISCUSSION TITLE
How to Move Page Numbers to the Right Bottom Corner in Quarto?

### DISCUSSION THREAD

**Discussion by stevefatz95** [quarto-dev/quarto-cli#10191] [Q&A] [answered]
### Description

Hi Quarto Community,

I'm currently working on a document and I would like to customize the placement of the page numbers. By default, the page numbers are centered at the bottom of the page, but I want to move them to the right bottom corner instead.

Could anyone guide me on how to achieve this in Quarto? Any specific YAML settings or LaTeX configurations that I need to adjust would be greatly appreciated.

Thank you for your help!

Best regards,

Stefano

**Comment by stevefatz95** [comment, marked as answer]
Hi everyone,

For those reading this thread, here is the solution.

Here is the YAML configuration to align the page number to the bottom right in Quarto:

```{yaml}
header-includes:
  - \usepackage{scrlayer-scrpage}
  - \usepackage{lipsum} % only for showing some sample text
  - \clearpairofpagestyles% remove all default elements from page styles scrheadings
                          % and plain.scrheadings
  - \ofoot*{\pagemark}% add pagination to the outer part of the page footer
  - \cfoot{}% clear the center part of the page footer
```

This configuration will help align the page number to the bottom right of the page.

**Reply by mcanouil** [reply]
You might want to use `include-in-header` instead of the Pandoc YAML option.

```yaml
format:
  pdf:
    include-in-header:
      - text: |
          \usepackage{scrlayer-scrpage}
          \usepackage{lipsum} % only for showing some sample text
          \clearpairofpagestyles% remove all default elements from page styles scrheadings and plain.scrheadings
          \ofoot*{\pagemark}% add pagination to the outer part of the page footer
          \cfoot{}% clear the center part of the page footer
```

> Here is the YAML configuration to align the page number to the bottom right in Quarto

For future readers, the question and answer is only about `format: pdf` (LaTeX).
